### ISSUE TITLE
Is there a gentler way to install erlang?

### ISSUE THREAD

**Issue by jdmarshall** [asdf-vm/asdf-erlang#120]
I'm running asdf install on an Armbian device and installing Erlang is so CPU intensive that it stops responding to ssh requests.  Seems like the parallelism might be a bit... aggressive.  It's doing a lot of post-download work and I'm wondering if that's particular to Armbian.  No precompiled binaries?  Is there a flag I can set to reduce the number of jobs?

I have a handful of these devices so if there's a way I can transfer from one machine to the others short of cloning the disks I'd be interested in any pointers.

**Comment by Stratus3D** [comment]
There is https://www.erlang-solutions.com/resources/download.html

I don't know if Erlang Solutions offers something that would work on Armbian, but I do see they offer Rasbian packages.

I suppose it would be nice to be able to give users the option of compiling from source or using a pre-built package.

**Comment by jdmarshall** [comment]
I don't believe it's the best optimization level, but it installs and seems to run.  

Is it a matter of not finding precompiled images for my machine or that it prefers installing from source?

**Comment by Stratus3D** [comment]
It prefers to install from source right now via kerl. We need to add support for install precompiled Erlang packages on our own if possible.

**Comment by drebits** [comment]
This worked for me: https://dev.to/drebits/how-to-install-erlang-on-macos-with-asdf-3p1c
